harness/uicore
# fix: [CDS-83279]: trim the extra whitespaces added before and after the value in new expression element

- removed the extra whitespace being added before and after the value in the new expression input element

Manually Tested cases :- 

`CHROME + FIREFOX` => 

1. Entering the expression value in the field without using the dropdown and verified the cursor position and value
2. Entering the expression value in the field using the dropdown and verified the cursor position and value
3. Using the local build of uicore in harness-core-ui and validating the same for runtime values in the expression field (video attached below)

CHROME -> 

https://github.com/harness/uicore/assets/106532291/8bc5190a-3605-4714-994e-55521dccaeff

FIREFOX -> 

https://github.com/harness/uicore/assets/106532291/787d22d6-9a4b-412b-9d5a-ff00cecc28f5

harness-core-ui => 


https://github.com/harness/uicore/assets/106532291/2c1e313b-afce-4ea1-a588-f2afe349b4f7





 
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`
